scripts/rollup/results.json

```diff
@@ -1,17 +1,17 @@
 {
-  "branch": "master",
+  "branch": "warnForKeysInFragments",
   "bundleSizes": {
     "react.development.js (UMD_DEV)": {
-      "size": 121454,
-      "gzip": 30515
+      "size": 121807,
```

Comment:
Actually I'm confused by how this works. When are we supposed to update this file? @gaearon @trueadm